mvn/archetypes/radix/src/main/resources/archetype-resources/crawler/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM openjdk:8-jre-alpine
+
+# Environment Variables (should not be modified)
+ENV OODT_HOME="/oodt"
+ENV CRAWLER_HOME="/oodt/crawler"
+
+# Environment Variables (should be user specified)
+ENV FAILURE_DIR=""
```

Comment:
@IMS94 just to clarify on this, these variables were set up because of the apps depending on them being available during execution. As an alternative, I can set them on the CMD itself, to avoid misconceptions. what do you suggest?

Reply:
You can't have them set home dirs and stuff because its container dependant, nor should it matter to the user.